### PR TITLE
Add status filter for get donations

### PR DESCRIPTION
**Pull request by FatemeGhasemi**
Related to #524

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
@RamRamez @CarlosQ96  After merging this PR we support status filter for `donationsByUserId` and `donationsByProjectId` queries, 
query samples:

* **donationsByProjectId**
 ```
  query (
    $take: Int
    $skip: Int
    $traceable: Boolean
    $projectId: Int!
    $searchTerm: String
    $status: String
    $orderBy: SortBy
  ) {
    donationsByProjectId(
      take: $take
      skip: $skip
      traceable: $traceable
      projectId: $projectId
      searchTerm: $searchTerm
      status: $status
      orderBy: $orderBy
    ) {
      donations {
        id
        transactionId
        transactionNetworkId
        toWalletAddress
        fromWalletAddress
        currency
        anonymous
        valueUsd
        amount
        status
        user {
          id
        }
        createdAt
      }
      totalCount
      totalUsdBalance
    }
  }
```
* **donationsByUserId**
```
 query (
    $take: Int
    $skip: Int
    $status: String
    $orderBy: SortBy
    $userId: Int!
  ) {
    donationsByUserId(
      take: $take
      skip: $skip
      orderBy: $orderBy
      userId: $userId
      status: $status
    ) {
      donations {
        id
        transactionId
        transactionNetworkId
        toWalletAddress
        fromWalletAddress
        currency
        anonymous
        valueUsd
        amount
        status
        user {
          id
        }
        project {
          id
        }
        createdAt
      }
      totalCount
    }
  }
```

File: src/resolvers/donationResolver.ts [hunk 1]
```diff
@@ -321,7 +330,8 @@ export class DonationResolver {
 
   @Query(returns => UserDonations, { nullable: true })
   async donationsByUserId(
-    @Args() { take, skip, orderBy, userId }: UserDonationsArgs,
+    @Args() { take, skip, orderBy, userId, status }: UserDonationsArgs,
+    // @Arg('status', type => String, { nullable: true }) status: string,
```
Comment (CarlosQ96) [inline src/resolvers/donationResolver.ts]: Please delete comment as this Argument is not used that way

**Review by CarlosQ96 — APPROVED** [review]
Remove the comment pls and lets merge. Seems good! Thanks Fatemi!

**Comment by mohammadranjbarz** [comment]
> @RamRamez @CarlosQ96 After merging this PR we support status filter for `donationsByUserId` and `donationsByProjectId` queries, query samples:
> 
> * **donationsByProjectId**
> 
> ```
>  query (
>    $take: Int
>    $skip: Int
>    $traceable: Boolean
>    $projectId: Int!
>    $searchTerm: String
>    $status: String
>    $orderBy: SortBy
>  ) {
>    donationsByProjectId(
>      take: $take
>      skip: $skip
>      traceable: $traceable
>      projectId: $projectId
>      searchTerm: $searchTerm
>      status: $status
>      orderBy: $orderBy
>    ) {
>      donations {
>        id
>        transactionId
>        transactionNetworkId
>        toWalletAddress
>        fromWalletAddress
>        currency
>        anonymous
>        valueUsd
>        amount
>        status
>        user {
>          id
>        }
>        createdAt
>      }
>      totalCount
>      totalUsdBalance
>    }
>  }
> ```
> 
> * **donationsByUserId**
> 
> ```
>  query (
>     $take: Int
>     $skip: Int
>     $status: String
>     $orderBy: SortBy
>     $userId: Int!
>   ) {
>     donationsByUserId(
>       take: $take
>       skip: $skip
>       orderBy: $orderBy
>       userId: $userId
>       status: $status
>     ) {
>       donations {
>         id
>         transactionId
>         transactionNetworkId
>         toWalletAddress
>         fromWalletAddress
>         currency
>         anonymous
>         valueUsd
>         amount
>         status
>         user {
>           id
>         }
>         project {
>           id
>         }
>         createdAt
>       }
>       totalCount
>     }
>   }
> ```

@RamRamez 
It's deployed on staging and you can use it now

**Pull request merged**